paimon-spark/paimon-spark-common/src/test/java/org/apache/paimon/spark/SparkReadITCase.java

```diff
@@ -704,4 +704,101 @@ public void testCreateNestedField() {
                                 "a INT",
                                 "b STRUCT<b1: STRUCT<b11: INT, b12: INT>, b2: BIGINT>"));
     }
+
+    @Test
+    public void testShowPartition() {
```

Comment:
sorry that forget to remind you in the first review, that we’d better to move this to `test/scala/org/apache/spark/sql/PaimonMangermentTest.scala`.

Reply:
after this, we can use `T` to replace `partition_table_01` or `drop_partition_table_01` table name, that will be dropped automatically when this UT finishes.